onnx/reference/ops/_helpers.py

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: Apache-2.0
+from onnx.reference.op_run import OpRun, _split_class_name
+
+
+def _build_registered_operators_any_domain(clo, load_op):  # type: ignore
+    reg_ops = {}  # type: ignore
```

Comment:
The dictionary seems slightly complicated which may benefit from being typed

Reply:
done